Cabal/Distribution/Simple/Build.hs

```diff
@@ -406,8 +407,9 @@ testSuiteLibV09AsLibAndExe pkg_descr
     libClbi = LibComponentLocalBuildInfo
                 { componentPackageDeps = componentPackageDeps clbi
                 , componentPackageRenaming = componentPackageRenaming clbi
-                , componentLibraries = [LibraryName (testName test)]
+                , componentLibraryName = LibraryName "test"
```

Comment:
Since we can have multiple test suites in the same package, won't this cause problems?